src/lib/project.js

```diff
@@ -390,36 +382,41 @@ const __dirname = path.dirname(__filename);
 };`
   );
 
-  // This prevents usEffect from running twice on initial mount.
-  newNextConfig = newNextConfig.replace(
-    'reactStrictMode: true',
-    'reactStrictMode: false'
-  );
-
   fs.writeFileSync(path.join('ui', 'next.config.mjs'), newNextConfig);
 
-  const indexFileName = useTypescript ? 'index.tsx' : 'index.js';
+  const pageFileName = 'page.tsx';
+
```

Comment:
It looks like there are some legacy `create-next-app` boilerplate styles and fonts leftover in the `app` folder when a project is generated. I think it is a good idea to remove these so they do not conflict with the other assets. You could empty the `app` directory with something like `fs.emptyDirSync`  or similar before adding the customNextPage and any other files to the directory.

Reply:
Good catch, cleared out the whole directory